### PR TITLE
Add Ada/SPARK

### PR DESCRIPTION
**Pull request by ohenley**
Where:
**https://github.com/ohenley/awesome-ada#readme**

What:
**A curated list of awesome resources related to the Ada and SPARK programming language**

Comments over other pull requests:
**https://github.com/sindresorhus/awesome/pull/1472#issuecomment-448830383**
**https://github.com/sindresorhus/awesome/pull/1457#issuecomment-448828524**

### By submitting this pull request I confirm I've read and complied with the below requirements 🖖

**Please read it multiple times. I spent a lot of time on these guidelines and most people miss a lot.**

## Requirements for your pull request

- **You have to review at least 2 other [open pull requests](https://github.com/sindresorhus/awesome/pulls?q=is%3Apr+is%3Aopen).** Try to prioritize unreviewed PRs, but you can also add more comments to reviewed PRs. Go through the below list when reviewing. This requirement is meant to help make the Awesome project self-sustaining. Comment here which PRs you reviewed. You're expected to put a good effort into this and to be thorough. Look at previous PR reviews for inspiration.
- I have read and understood the [instructions for creating a list](https://github.com/sindresorhus/awesome/blob/master/create-list.md).
- This pull request has a descriptive title in the format `Add Name of List` (Example: `Add Swift`), not `Update readme.md` or `Add awesome list`.
- The entry in the Awesome list should:
	- Include a short description about the project/theme of the list. **It should not describe the list itself.**<br>Example: `- [Fish](…) - User-friendly shell.`, not `- [Fish](…) - Resources for Fish.`.
	- Be added at the bottom of the appropriate category.
- The list I'm submitting complies with the below requirements.

## Requirements for your Awesome list

- **Has been around for at least 30 days.**<br>That means 30 days from either the first real commit or when it was open-sourced. Whatever is most recent.
- It's the result of hard work and the best I could possibly produce.
- Non-generated Markdown file in a GitHub repo.
- **Includes a succinct description of the project/theme at the top of the readme.** [(Example)](https://github.com/willempienaar/awesome-quantified-self)
- The repo should have `awesome-list` & `awesome` as [GitHub topics](https://help.github.com/articles/about-topics). I encourage you to add more relevant topics.
- Not a duplicate.
- Only has awesome items. Awesome lists are curations of the best, not everything.
- Includes a project logo/illustration whenever possible.
	- Either centered, fullwidth, or placed at the top-right of the readme. [(Example)](https://github.com/sindresorhus/awesome-electron)
	- The image should link to the project website or any relevant website.
	- The image should be high-DPI. Set it to maximum half the width of the original image.
- Entries have a description, unless the title is descriptive enough by itself. It rarely is though.
- Includes the [Awesome badge](https://github.com/sindresorhus/awesome/blob/master/awesome.md#awesome-badge).
	- Should be placed on the right side of the readme heading.
		- Can be placed centered if the list has a centered graphics header.
	- Should link back to this list.
- Has a Table of Contents section.
	- Should be named `Contents`, not `Table of Contents`.
	- Should be the first section in the list.
	- Should only have one level of [nested lists](https://commonmark.org/help/tutorial/10-nestedLists.html), preferably none.
- Has an [appropriate license](https://github.com/sindresorhus/awesome/blob/master/awesome.md#choose-an-appropriate-license).
	- That means something like CC0, **not a code licence like MIT, BSD, Apache, etc.**
	- [WTFPL](http://www.wtfpl.net) and [Unlicense](http://unlicense.org) are not acceptable licenses.
	- If you use a license badge, it should be SVG, not PNG.
- Has [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/awesome.md#include-contribution-guidelines).
	- The file should be named `contributing.md`. Casing is up to you.
- Has consistent formatting and proper spelling/grammar.
	- The link and description are separated by a dash. <br>Example: `- [AVA](…) - JavaScript test runner.`
	- The description starts with an uppercase character and ends with a period.
	- Consistent and correct naming. For example, `Node.js`, not `NodeJS` or `node.js`.
- Doesn't include a Travis badge.<br>You can still use Travis for list linting, but the badge has no value in the readme.

Go to the top and read it again.

### CODE REVIEW

**Comment by sindresorhus** [comment]
The header looks awesome!

--

Some minor nitpick:

```
  ✖    8:2    Add 1 space between blockquote and content                  blockquote-indentation
  ✖  238:63   List item description must end with proper punctuation      awesome/list-item
  ✖  256:46   List item description must end with proper punctuation      awesome/list-item
  ✖  296:72   List item description must start with valid casing          awesome/list-item
  ✖  315:49   List item description must end with proper punctuation      awesome/list-item
  ✖  320:3    Link to http://stephe-leake.org/ada/opentoken.html is dead  no-dead-urls

File: readme.md [hunk 1]
```diff
@@ -177,6 +177,7 @@
 - [ActionScript 3](https://github.com/robinrodricks/awesome-actionscript3#readme) - Object-oriented language targeting Adobe AIR.
 - [Eta](https://github.com/sfischer13/awesome-eta#readme) - Functional programming language for the JVM.
 - [Idris](https://github.com/joaomilho/awesome-idris#readme) - General purpose pure functional programming language with dependent types influenced by Haskell and ML.
+- [Ada/SPARK](https://github.com/ohenley/awesome-ada#readme) - Ada/SPARK is a modern programming language designed for large, long-lived applications where reliability and efficiency are essential.
```
Comment (sindresorhus) [inline readme.md]: Seeing as the list is called Awesome Ada, I don't think the title `Ada/SPARK` is correct here.

Comment (ohenley) [reply readme.md]: I do not get it. 

Python, Javascript, Swift, Rust, Go, Idris (do not have time to check the countless others) ... they are all called 'awesome-XYZ' and listed only as their plain name XYZ? 

eg -> [Idris](https://github.com/joaomilho/awesome-idris#readme)

Ada/SPARK is a programming language and it is its name.

**Comment by ohenley** [comment]
I fixed the 6 changes.
https://github.com/ohenley/awesome-ada/commit/524015dbbba136bc5c924465dd76984d81e1dcb0

thx for the great feedback.

File: readme.md (hunk 1 above)
Comment (sindresorhus) [reply readme.md]: I'm not talking about the `awesome-` prefix. I'm just confused what the actual name is. The name written here is `Ada/SPARK`. The PR title is just `Ada`. The repo name is `awesome-ada`, not `awesome-ada-spark`. In your list it's written as both `Ada`, `Ada/SPARK`, `SPARK/Ada`. So which is it? You should be consistent about the naming everywhere.

Comment (ohenley) [reply readme.md]: Ah ok. SPARK is a recent 'subset' of Ada but of great importance. That is why usually people refer to one or the other, like I did. The core stays Ada but SPARK is a major addition for its formal verification and it is really tied to the latest Ada standard (2012). It is a bit of particular situation if you want my opinion.

Anyway, I renamed the PR to match the description and my list description refers to Ada/SPARK too. If I am not mistaken, things should are consistent now.

Thank you for your time and concern.

**Comment by sindresorhus** [comment]
Tweet: https://twitter.com/awesome__re/status/1096832842766606337

**Pull request merged**